sftp/sftp.tf

```diff
@@ -7,34 +7,77 @@ terraform {
   }
 }
 
+provider "aws" {
+  alias = "ephemeral"
```

Comment:
From the point of view of this repo/module, is this really an 'ephemeral' vs 'permanent' configuration? What is actually being achieved by this replication? The naming should reflect that. I think we're leaking Honest concepts and business logic into this.

In the readme we mention:
> * Backs up the SFTP server S3 bucket to another bucket

However this is not reflected in the naming or anything (which uses the terminology 'ephemeral' and 'permanent' instead of something like 'backup_source' and 'backup_target' or whatever the industry standard terminology for backups are.